Library/Homebrew/os.rb

```diff
@@ -25,7 +25,7 @@ def self.linux?
     PATH_OPEN = "/usr/bin/open".freeze
   elsif OS.linux?
     require "os/linux"
-    ISSUES_URL = "https://github.com/Linuxbrew/brew/wiki/troubleshooting".freeze
+    ISSUES_URL = "https://docs.brew.sh/Troubleshooting".freeze
```

Comment:
This URL is now identical for both macOS and and Linux. It can be moved outside the `if` or to `globals.rb`.

Reply:
Not quite; it's not set unconditionally for macOS but it is on Linux.

Reply:
> `docs/Homebrew-on-Linux-and-Windows-10.md` This slug is long 
> it's necessary to be unnecessarily verbose in the slug. 

A longer slug is better for SEO and we want the slug to match the page title. 

> and `Windows-10` is not future proof when Windows 11 comes out. 

From what I have read there will never be a Windows 11 and YAGNI. If that did happen: we can redirect.

Reply:
I'd suggest
`docs/Homebrew-on-Linux-and-Windows.md` or
`docs/Homebrew-on-Linux-and-WSL.md`.
Can `https://brew.sh/docs/Linux` redirect to the full name?

Reply:
`Windows-10-WSL` or `Windows-10` from my perspective as it's not Windows in general.



> Can `https://brew.sh/docs/Linux` redirect to the full name?

Yup, will do.

Reply:
`docs/Homebrew-on-Linux.md` ?